include/configclass.php

```diff
@@ -1244,8 +1244,9 @@ function setSvnCommandPrefix($prefix) {
 	function _updateSVNCommand() {
 		$this->_setPath($this->svn, $this->_svnCommandPath, 'svn', '--non-interactive --config-dir '.$this->_svnConfigDir.($this->_svnTrustServerCert ? ' --trust-server-cert' : ''));
 		$this->_setPath($this->svnAuthz, $this->_svnCommandPath, 'svnauthz', 'accessof');
-		$this->svn = $this->_svnCommandPrefix.' '.$this->svn;
-		$this->svnAuthz = $this->_svnCommandPrefix.' '.$this->svnAuthz;
+
+		$this->svn		= trim($this->_svnCommandPrefix.' '.$this->svn);
+		$this->svnAuthz	= trim($this->_svnCommandPrefix.' '.$this->svnAuthz);
```

Comment:
Why are these trims necessary?

Reply:
Because `_svnCommandPrefix` is empty most likely and then additional spaces are introduced which make quoting the command unnecessary difficult. Quoting was the first implemented approach and might come back for some reason, so not forwarding unnecessary spaces to shells seems to be the correct thing to do.

Besides that, I don't see a reason for `_svnCommandPrefix` at all right now: It's not documented in `distconfig.php`, only used in the place above and such an approach simply adding textual prefixes to things forwarded to shells is barely portable to different OSes. How broken this is can be seen already by bypassing `_setPath`, so I removed it entirely.

https://github.com/websvnphp/websvn/commit/6fc0f344c1823669f3b05b36cb0a799cac910d45#diff-ca4966c9756196b9fbf141a353fab2feR1150